### DISCUSSION TITLE
What does all the icons in front of each commit (in commit view, as well as in branches view) mean?

### DISCUSSION THREAD

**Discussion by aDisplayName** [gitkraken/vscode-gitlens#1420] [Q&A]
![image](https://user-images.githubusercontent.com/19527258/110761885-32d7ca80-828b-11eb-91c1-a0b6a0a01238.png)
Is there some explanation somewhere?

Much thanks.

**Comment by eamodio** [comment]
Those are the author avatars, using Gravatar. If the user doesn't have an avatar setup there Gravatar serves up a default set.
